src/lib/tvgFill.cpp

```diff
@@ -43,11 +43,11 @@ Fill::~Fill()
 
 Result Fill::colorStops(const ColorStop* colorStops, uint32_t cnt) noexcept
 {
```

Comment:
I think it must return immediately after checking invalidArguments.
ie)
if ((!colorStops && cnt > 0) || (colorStops && cnt ==0)) return invalid arguments;

Reply:
colorStops(nullptr, 4) -> invalid argument
colorStops(ptr, 0) -> invalid argument 
colorStops(nullptr, 0) -> erease 

above is ok?